pandas/tests/sparse/frame/test_to_from_scipy.py

```diff
@@ -49,21 +47,19 @@ def test_from_to_scipy(spmatrix, index, columns, fill_value, dtype):
         fill_value if fill_value is not None else np.nan)
 
     # Assert frame is as expected
-    sdf_obj = sdf.astype(object)
+    # what is this test?
+    sdf_obj = sdf.astype(SparseDtype(object, fill_value))
```

Comment:
was it needed to change this? (I would think `astype(object)` should still do the same as before?)